cmd/server/app/options/options.go

```diff
@@ -292,17 +291,8 @@ func (o *ProxyRunOptions) Validate() error {
 	}
 
 	// validate the proxy strategies
-	if o.ProxyStrategies != "" {
-		pss := strings.Split(o.ProxyStrategies, ",")
-		for _, ps := range pss {
-			switch ps {
-			case string(server.ProxyStrategyDestHost):
-			case string(server.ProxyStrategyDefault):
-			case string(server.ProxyStrategyDefaultRoute):
-			default:
-				return fmt.Errorf("unknown proxy strategy: %s, available strategy are: default, destHost, defaultRoute", ps)
-			}
-		}
+	if _, err := server.ParseProxyStrategies(o.ProxyStrategies); err != nil {
```

Comment:
Might be nice to explicitly handle the empty case and indicate that a proxy strategy is required.

Reply:
Done.